server/Dockerfile

```diff
@@ -6,13 +6,14 @@ VOLUME /timely-server-src
 VOLUME /timely
 
 RUN yum -y update && \
-  yum -y install java-1.8.0-openjdk-devel maven openssl apr && \
-  mkdir -p /timely/bin
+  yum -y install java-1.8.0-openjdk-devel maven openssl apr wget && \
+  mkdir -p /timely/bin && \
+  wget -O /usr/lib/jvm/jre/lib/ext/bcprov-jdk15on-155.jar https://www.bouncycastle.org/download/bcprov-jdk15on-155.jar && \
+  echo "security.provider.9=org.bouncycastle.jce.provider.BouncyCastleProvider" >> /usr/lib/jvm/jre/lib/security/java.security
```

Comment:
Additional, user-configured providers should go in `/etc/java/security/security.d`. `/usr` doesn't typically hold  user-edited configs.